docs/src/pages/index.tsx

```diff
@@ -12,17 +27,155 @@ export default function Home() {
         css={css`
           /* so that the rest of the content has a higher z-order than the image */
           position: relative;
+          width: 100%;
+          max-width: 80rem;
+          margin: 0 auto;
+
+          > * + * {
+            margin-top: 5.25rem;
+          }
         `}
       >
-        <Heading type="display">
+        <div>
+          <Heading type="display">
+            <div
+              css={css`
+                max-width: 20ch;
+                font-size: 48px;
```

Comment:
noticed, somewhere in code you use `em` for font-size, why here `48px` and not `3em`?